### ISSUE TITLE
SSD 7 Pre-trained weights 

### ISSUE THREAD

**Issue by ManishSoni1908** [pierluigiferrari/ssd_keras#116]
Can you please provide the trained weight for SSD7 implementation ?
I am getting error when i am training from the scratch.

Thanks in advance

**Comment by pierluigiferrari** [comment]
Sorry, I don't provide trained weights for SSD7. If you elaborate on the error I'll try to help.

**Comment by ManishSoni1908** [comment]
Thanks for the reply,
Problem is that loss is not decreasing when i am training the model on cityscape dataset from scratch. Loss value starts from 37 and even after 40 epochs it was around 22 and it is just fluctuating for the further epochs.

**Comment by pierluigiferrari** [comment]
The model learns as expected on standard dataset like Pascal VOC or MS COCO, so the problem is almost certainly your dataset or the way you're feeding it to the model. If the loss doesn't decrease, that's usually an indicator that the model is receiving contradictory (i.e. non-sensical) ground truth.

Let's start with this: The Cityscapes dataset provides pixel-level class labels, it doesn't provide 2D bounding boxes. How did you produce the bounding boxes?
